README2.Md

```diff
@@ -0,0 +1,54 @@
+My approach on #Takeaway Challenge
```

Comment:
Really helpful for the review to have such clear instructions about how you worked on the challenge